lib/blueprinter/extractors/association_extractor.rb

```diff
@@ -5,7 +5,8 @@ def initialize
     end
 
     def extract(association_name, object, local_options, options={})
-      value = @extractor.extract(association_name, object, local_options, options.except(:default))
+      options_without_default = options.reject { |k,_| k == :default }
```

Comment:
Ahh nice catch. Would you mind bumping the version of Blueprinter for a patch fix?
https://github.com/procore/blueprinter/blob/master/lib/blueprinter/version.rb#L2
Would become `VERSION = 0.12.2`.
And then add an entry to the changelog? 
https://github.com/procore/blueprinter/blob/master/CHANGELOG.md